react/.env.development

```diff
@@ -1,4 +1,2 @@
-# disable opening browser on start
-BROWSER=none
 HOST=127.0.0.1
 PORT=3081
```

Comment:
Please change the port number so that the developer can connect to the same port as before.

Reply:
Also, did you test on a remote environment such as vmfarm?

Reply:
Additionally, let's update README.md to guide how to change the port and host for the remote environment

Reply:
When ready to review again, please move #1799 to the "In review" column on the Daily board